audit/projects/kubernetes-public/services/compute/project-info.json

```diff
@@ -5,6 +5,10 @@
         "key": "gke-aaa-4d0eb763-secondary-ranges",
         "value": "services:default:gke-aaa-subnet-4d0eb763:gke-aaa-services-4d0eb763,pods:default:gke-aaa-subnet-4d0eb763:gke-aaa-pods-4d0eb763"
       },
+      {
+        "key": "ssh-keys",
+        "value": "arnaudm:ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDIQxCxqIeW9zArVuqZ7WhkURWkcQdj+PVqzTdUb65joIxS0hWnYR6gpKtKOUlLw+YUUuAAvoUjl2MBNHU8R1ctJ7V0ISf6IEQuFZa7aS68bYy92gzH1QDpgHeyHlFbOrzZbLFWjebnKlT2wQL+29JB/5oYwY6RW1a+vas0rI6GiHLUu5rEUuLr38lGni9rhoQcidcDtEG7rG/nfa64ZrhDVeiUt0udmZaViXgrlLLTaJjnZTQ1sI5IuG42EEBpGDhHRkDwhAiLNjjVVkoBWDtPMyT/WbraoqShPzBMKhJz6NtS61cF2yMWIN+xZbJoPDJCcwzvQ9sYlcYrt2LEn75UEN+554lrBeQHjdwumiKjAzR/4m+kUHR9nm+a0li5TJUAmQ5K3pKD6ju2xcyrtzaQQ48FJm0y7fIET5dl4fQgLPj8hD9p/UwETF+9lV/XMH0EEyh0AoFWt/X/oDzH9/eBo7bn03Lugj3eYlvM1griu3OB8Iz5HA3oRw0H6JtdX7U= ameukam@barbatos"
```

Comment:
Needed to access to the Cloud SQL instances for `elections.k8s.io` from a GCE instance.